pkg/policy/custompolicybuilder/custom-policy-manager.go

```diff
@@ -0,0 +1,120 @@
+package custompolicybuilder
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+type PolicyTemplate struct {
+	Name      string
+	CheckType string
+	Dir       string
+	Type      string
+	Desc      string
+	Title     string
+	Severity  string
+	Category  string
+	RsrcType  string
+}
+
+func (pt *PolicyTemplate) PromptInput() error {
+	var qs = []*survey.Question{
+		{
+			Name: "dir",
+			Prompt: &survey.Input{
+				Message: "Policies directory path",
+				Default: "policies"},
+			Validate: pt.validatePolicyDirectory(),
+		},
+		{
+			Name: "checkType",
+			Prompt: &survey.Select{
+				Message: "Select target:",
+				Options: []string{"terraform", "cloudformation", "kubernetes", "arm"},
+			},
+		},
+		{
+			Name: "name",
+			Prompt: &survey.Input{
+				Message: "policy name",
+				Help:    "Example policy name: my_policy_1"},
+			Validate: pt.validatePolicyName(),
+		},
+		{
+			Name:   "title",
+			Prompt: &survey.Input{Message: "Title"},
+		},
+		{
+			Name:   "desc",
+			Prompt: &survey.Input{Message: "Description"},
+		},
+		{
+			Name:   "category",
+			Prompt: &survey.Input{Message: "Category"},
+		},
+		{
+			Name:   "rsrcType",
+			Prompt: &survey.Input{Message: "ResourceType"},
```

Comment:
Without any examples, the users might not understand when we say resourceType.

Reply:
Added example